src/Microsoft.DotNet.Darc/src/DarcLib/Actions/Local.cs

```diff
@@ -165,5 +166,17 @@ public void AddRemoteIfMissing(string repoDir, string repoUrl)
         {
             _gitClient.AddRemoteIfMissing(repoDir, repoUrl);
         }
+
+        private List<GitFile> GetFilesAtRelativeRepoPathAsync(string path)
```

Comment:
It'd be good to have this method covered by unit tests

Reply:
It was just moved in this PR, I'd leave adding tests for the debt epic honestly